### PR TITLE
v0.3.0: prepare new release

### PR DESCRIPTION
**Pull request by behackl**
Preparations for new release:

- Version number bump (to v0.3.0)
- Include list of changes since v0.2.0

For the sake of documentation, here is what I did to get to this changelog:

1. `git log --format="- %s" --first-parent 8a673946.. > changelog`, this creates a file `changelog` with one line per commit on master
2. Regex-replacement: 
   ```
   - (.+) \(#(\d+)\)$
   ```
   to
   ```
   - :pr:`$2`: $1
   ```
3. Fix possibly remaining `Merge pull request ...` from non-squish merges and make them consistent with other entries
4. Go through the list from bottom to top, classify into `Breaking Changes`, `New Features`, `Bugfixes and Enhancements`, some messages were prefixed by `Bugfix:`, etc. -- delete these in favor of placement in correct category.
5. Look through list and add references/code literals wherever it makes sense.

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -17,6 +17,71 @@ Upcoming release
...
+- :pr:`922`: Tree layout for Graph mobject
+- :pr:`935`: Added code of conduct
+- :pr:`916`: Multi-column layout for partite graphs
+- :pr:`742`: Units: Pixels, Munits, Percent in :mod:`~.utils.unit`
+- :pr:`893`: Convenience method :meth:`~.Graph.from_networkx` for creating a graph from a networkx graph
+
+Bugfixes and Enhancements
+=========================
+
+- :pr:`961`: added typings and docs for vectorized mobjects and bezier related functions
```
Comment (WampyCakes) [inline docs/source/changelog.rst:49]:
```suggestion
- :pr:`961`: Added typings and docs for vectorized mobjects and bezier related functions
```

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -17,6 +17,71 @@ Upcoming release
...
+- :pr:`936`: Fix :class:`~.BulletedList` TeX environment problem and add a typing to ``get_module``
+- :pr:`938`: Remove dependency on progressbar
+- :pr:`937`: Change 'brew cask install' to 'brew install --cask' for CI pipeline
+- :pr:`933`: Make matrix work with lists again
+- :pr:`932`: Correctly parse ``log_dir`` option
+- :pr:`920`: Raise error if markup in :class:`~.MarkupText` is invalid
+- :pr:`929`: Raise an error if a :class:`~.Matrix` object is created with < 2-dimensional input
+- :pr:`907`: Make Scene.add_sound work again (when running with ``--disable_caching``)
+- :pr:`906`: Allow new-style method animation to be used in animation groups
+- :pr:`908`: removed deprecated command line arguments from documentation
```
Comment (WampyCakes) [inline docs/source/changelog.rst:79]:
```suggestion
- :pr:`908`: Removed deprecated command line arguments from documentation
```

File: pyproject.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "manim"
-version = "0.2.0"
+version = "0.2.1"
```
Comment (eulertour) [inline pyproject.toml:3]: Shouldn't this be 0.3.0? We didn't do a mid-month release.

Comment (jsonvillanueva) [reply pyproject.toml:3]: If we're using semantic versioning, then points 4, 6 and 7 are [useful references](https://semver.org/#spec-item-4). I think this release adds a few new backwards compatible features and would justify bumping the minor instead. Perhaps future month releases will only involve bug fixes/patches, but I foresee **many** minor bumps as we add new features each month to the core (I'm calling it here, 0.11.2 -> 1.0.0)

Comment (behackl) [reply pyproject.toml:3]: As long as our major version is still 0, that doesn't really apply though necessarily. That being said, I'd be fine with bumping to v0.3.0 (although I was thinking of moving to v0.3.0 only with the switch to the webgl renderer).

**Comment by naveen521kk** [comment]
@behackl Do you have any ideas of using https://github.com/twisted/towncrier?

**Comment by behackl** [comment]
> @behackl Do you have any ideas of using https://github.com/twisted/towncrier?

No, and I don't really see a reason for using it.

File: pyproject.toml (hunk 3 above)
Comment (behackl) [reply pyproject.toml:3]:
```suggestion
version = "0.3.0"
```

Comment (behackl) [reply pyproject.toml:3]: From the discussion on Discord it seems that people are either neutral or leaning towards `v0.3.0` -- so let's adhere to semantic versioning, respect that we've introduced new, backwards-compatible features, and let's go with `v0.3.0`. I'm pushing another commit.

**Review by eulertour — APPROVED** [review]

**Pull request merged**